tests/conftest.py

```diff
@@ -19,4 +19,25 @@ def ruff_version() -> Version:
         workspace=uris.from_fs_path(workspace_path),
     )
 
-    return _find_ruff_binary(settings, version_requirement=None).version
+    return _find_ruff_binary(settings, version_requirement=None)
+
+
+@pytest.fixture(scope="session")
+def ruff_version() -> Version:
+    return _get_ruff_executable().version
+
+
+def pytest_report_header(config):
+    """Add ruff version to pytest header."""
+    executable = _get_ruff_executable()
+
+    # Display the long version if the executable supports it
+    try:
+        output = subprocess.check_output([executable.path, "version"]).decode().strip()
+    except subprocess.CalledProcessError:
+        output = (
+            subprocess.check_output([executable.path, "--version"]).decode().strip()
+        )
```

Comment:
We use this instead of simply `executable.version` so we can get richer information when available.